internal/pkg/cli/env_init.go

```diff
@@ -285,7 +285,7 @@ func (o *initEnvOpts) Execute() error {
 	// If the call fails because the role already exists, nothing to do.
 	// If the call fails because the user doesn't have permissions, then the role must be created outside of Copilot.
 	_ = o.iam.CreateECSServiceLinkedRole()
-	
+
```

Comment:
I thought we'd accept 
```
$ copilot app init --permissions-boundary A
$ copilot env init --permissions-boundary B
```
and we'd use boundary policy B - instead of A - for all the roles within the environment.

Is this going to be in an upcoming PR, or did I misunderstand?

Reply:
Also I wonder what'd be the story for `copilot init`? Do we want to accept this flag there?

Reply:
> and we'd use boundary policy B - instead of A - for all the roles within the environment.
>
> Is this going to be in an upcoming PR, or did I misunderstand?

We won't do this until we hear a customer request :) 

> Also I wonder what'd be the story for copilot init? Do we want to accept this flag there?

My opinion is that this feature is for L+ organizations and doesn't fit into `init` which is more for just getting started.